kore.maude

```diff
@@ -220,18 +227,30 @@ Questions about this definition:
 - do we really need ~> as a K construct?  Can it be a label and thus the way K
   deal with evaluation contexts just a particular methodology and not an
   intrisic part of K ? Same question for =>.
+  Cosmin: Syntacticly at least, I would say "no" for both. From the implementation
+  perspective, this will simplify traversal code even further. 
 - rename .::K and .::KList to . ?  if we get rid of ~> as above, then we may
   only need .::KList.
+  Cosmin: I agree.
 - Should we allow empty productions instead of sort declarations?  A pro
   argument is that we have fewer syntactic constructs.  A cons argument is
   that a sort declaration may be regarded as a different entityr, with
   different kinds of attributes.
+  Cosmin: I don't quite follow.
```

Comment:
I don't quite like this because `syntax Sort ::=   [Att]` make it seem like this is a sort for nothing.

I have also been thinking of eliminating sort declarations, but not by replacing them with productions but by replacing them to with subsorts. Instead of writing:
```
syntax Foo
```
it would be
```
syntax Top ::= Foo
```
As I said before, my preference would to be have a special `sort` sentence and be able to do:
```
sort Foo < Top
```
I understand we may want to have all sorts represented as undeleted injections, but I think we're very far from there and introducing a `sort` sentence would clarify things in the meantime. It will also make it clearer if we represent sorts as injections. We would know that if we eliminate all the `sort` sentences we are there.